feature/kanban_board/client/components/App.jsx

```diff
@@ -1,23 +1,91 @@
 /* eslint-disable quotes */
 import React, { Component } from 'react';
 import '../assets/stylesheets/style.scss';
-import { withRouter } from 'react-router-dom';
 import { connect } from 'react-redux';
+import Backend from 'react-dnd-html5-backend';
+import {DndProvider} from 'react-dnd';
+import {fetchColumnList , createColumn} from "../redux/actions/columnActions";
+import Column from './Column';
 
 class App extends Component {
   constructor(props) {
     super(props);
+    this.state={
+      columnTitle: ''
+    }
   }
 
+  componentDidMount(){
+    this.props.fetchColumnList(this.createColumnList)
+  }
+  
+  createColumnList = () => {
+    console.log("inside callback")
```

Comment:
Please do not leave `console` statements in code.

Reply:
resolved